packages/fast-components-styles-msft/src/radio/index.ts

```diff
@@ -53,28 +59,18 @@ const styles: ComponentStyles<RadioClassNameContract, DesignSystem> = (
             height: "20px",
             appearance: "none",
             borderRadius: "50%",
-            boxSizing: "content-box",
             margin: "0",
             zIndex: "1",
-            background: backgroundColor,
-            boxShadow: `inset 0 0 0 1px ${radioColor}`,
+            ...outlinePattern.rest,
             "&:hover": {
-                boxShadow: `inset 0 0 0 1px ${radioHover}`,
+                ...outlinePattern.hover,
```

Comment:
![image](https://user-images.githubusercontent.com/37851214/49839759-630b2880-fd65-11e8-9edf-b6291da42df4.png)
Little bit of strange aliasing in Dark theme with the focus outline.